### ISSUE TITLE
Consider moving accessibility support setting under the workbench namespace

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#42990]
Currently we have `editor.accessibilitySupport` which can set by triggering the "Show Accessibility Help" command and hitting ctrl/cmd+e. The issue is that this can only be set when an editor is focused, the badge also only appears when an editor is open and the setting doesn't show up in settings intellisense.

The terminal now leverages this setting to determine whether to do the extra a11y work, I think it would make sense to pull the setting into the `workbench.` namespace, always display the badge in the status bar and fix intellisense in the settings.

---

There are also issues with the display currently due to it being confined within an editor:

<img width="495" alt="screen shot 2018-02-05 at 11 01 41 am" src="https://user-images.githubusercontent.com/2193314/35823335-0965f34a-0a64-11e8-8d06-0973429661ef.png">

**Comment by alexandrudima** [comment]
@Tyriar it does not matter that the text is having layout issues in tiny editors, that text is there only for Screen Readers.

---

Please feel free to convert it to a workbench modal dialog. The code currently sits here: `src/vs/workbench/parts/codeEditor/electron-browser/accessibility.ts` and does not affect the standalone editor, so it can be moved/transformed to something else.

There is just this flow we should preserve:
> If `editor.accessibilitySupport` is set to `off`, it should be possible to use a screen reader and reach this panel (via information presented to the screen reader), and the panel should help to toggle the setting back to `on`.

**Comment by bpasero** [comment]
Help Wanted, really?
